content/features/search-filters.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Searching on Crunch (just got better)"
+description = "Use filters to refine your searches and find what you are looking for faster"
+date = "2017-07-06T12:59:58-04:00"
+weight = 20
+draft = false
+tags = []
+categories = ["general", "search"]
++++
+
+Search has been a part of Crunch from the beginning. In a system designed to house and share large volumes of data, the ability to quickly find the data that's important right now is critical. As Crunch has grown, we've worked hard to continually improve how data is indexed to make searching faster and more accurate. Even as we continue these efforts, we are happy to announce that we've added a couple new ways for you to refine your searches to get to what you are looking for faster and more easily.
+
+## Search Filters
+
+Two dropdowns at the top of the search panel help you to dial in your search.
+
+
+{{< figure src="../images/SearchTypeFilter.png" class="floating-left">}}
+
+On the left, the **Type filter** lets you specify **Datasets**, **Variables**, or **Categories** to limit your search to that type of object. For example, imagine you had a dataset titled "Changing attitudes towards Gender Identity"; you might try searching for it using the term **Gender**, only to find that you get dozens of results from every dataset with a "Gender" variable. By limiting the search to **Datasets**, you can quickly find what you are looking for.

```

Comment:
The screenshot doesn't match this text. There's no "Datasets" option in dropdown, presumably because it's searching within a dataset?